packages/docs/src/examples/v-rating/slot-item.vue

```diff
@@ -7,7 +7,7 @@
           large
           @click="props.onClick"
         >
-          {{ props.isFilled ? 'mdi-star-circle' : 'mdi-star-circle-outline' }}
+          mdi-star-circle{{ props.isFilled ? '' : '-outline' }}
```

Comment:
not really worth arguing over, but I think that for an example, the old way is actually better/clearer

Reply:
I'm in agreeance, convert to the previous implementation.